setup.py

```diff
@@ -83,6 +82,7 @@ def get_version():
         'scripts/cfn2py'
     ],
 
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
```

Comment:
I'm fine with removing testing and mention of 3.4 but I'm not sure I want to explicitly prevent use on 3.4...or at least until it is officially EOL. Could you adjust this to remove 3.4 for now? Thanks.

Reply:
Done